index.html

```diff
@@ -1608,7 +1608,325 @@ <h3>Defining 'string'</h3>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+2006 SIX-PER-EM SPACE</span></td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+2007 FIGURE SPACE</span></td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+2008 PUNCTUATION SPACE</span></td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+2009 THIN SPACE</span></td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+200A HAIR SPACE</span></td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+200E LEFT-TO-RIGHT MARK</span></td>
+					<td>&nbsp;</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+200F RIGHT-TO-LEFT MARK</span></td>
+					<td>&nbsp;</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+2028 LINE SEPARATOR</span></td>
+					<td>✓</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+2029 PARAGRAPH SEPARATOR</span></td>
+					<td>✓</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+202F NARROW NO-BREAK SPACE</span></td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+205F MEDIUM MATHEMATICAL SPACE</span></td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+3000 IDEOGRAPHIC SPACE</span></td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+				</tr>
+				<tr>
+					<td><span class="uname" translate="no">U+FEFF ZERO WIDTH NO-BREAK SPACE</span></td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>&nbsp;</td>
+					<td>✓</td>
+					<td>&nbsp;</td>
+				</tr>
+			</tbody>
+		</table>
+		<p>
```

Comment:
Close up whitespace and make this paragraph a single line.

Reply:
Done.